<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1200</width>
      <height>930</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="predict">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="Label_img_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>130</y>
        <width>160</width>
        <height>300</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Button_save_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>평가 1 저장</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_save_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>평가 2 저장</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>430</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>검사 제품 이미지</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Label_img_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>510</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>810</y>
        <width>340</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>모델 선택</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>810</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>평가2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>정상 제품 이미지</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Label_img_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>491</width>
        <height>90</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Label_img_name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>350</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Botton_img_setting">
       <property name="geometry">
        <rect>
         <x>359</x>
         <y>20</y>
         <width>110</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>검사 이미지 지정</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="Label_img">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>470</y>
        <width>340</width>
        <height>240</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Button_reset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>140</width>
         <height>200</height>
        </rect>
       </property>
       <property name="text">
        <string>초기화</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_open_directory">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>140</width>
         <height>200</height>
        </rect>
       </property>
       <property name="text">
        <string>저장 폴더 열기</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>130</y>
        <width>160</width>
        <height>300</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Button_run_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>제품 평가 1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_run_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>제품 평가 2</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>810</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>평가1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>600</width>
        <height>90</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Label_directory">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>450</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_directory">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>20</y>
         <width>110</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>저장 폴더 지정</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>730</y>
        <width>340</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="comboBox_model">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="Label_model">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Button_model_reset">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>820</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>새로고침</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="train">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>600</y>
        <width>535</width>
        <height>50</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_progress">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_7">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>500</width>
        <height>570</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>450</width>
         <height>50</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>100</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>File Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>200</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>450</width>
         <height>50</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>100</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Image Width</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_width">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>200</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>450</width>
         <height>50</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>100</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Image Height</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_height">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>200</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>450</width>
         <height>50</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>100</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Epoch</string>
        </property>
       </widget>
       <widget class="QSlider" name="slider_epoch">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>200</width>
          <height>30</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="Label_slider_value">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>0</y>
          <width>100</width>
          <height>50</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>400</y>
         <width>450</width>
         <height>50</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="Button_image_path">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Image Path</string>
        </property>
       </widget>
       <widget class="QLabel" name="Label_image_path">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>300</width>
          <height>30</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>490</y>
         <width>450</width>
         <height>50</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="Button_save_path">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Path</string>
        </property>
       </widget>
       <widget class="QLabel" name="Label_save_path">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>300</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="Button_model_creation">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>760</y>
        <width>240</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Creation</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
